src/Compilers/VisualBasic/Portable/Lowering/Instrumentation/CompoundInstrumenter.vb

```diff
@@ -0,0 +1,262 @@
+            Return Previous.InstrumentSelectStatementEpilogue(original, epilogueOpt)
+        End Function
+
+        Public Overrides Function InstrumentFieldOrPropertyInitializer(original As BoundFieldOrPropertyInitializer, rewritten As BoundStatement, symbolIndex As Integer, createTemporary As Boolean) As BoundStatement
+            Return Previous.InstrumentFieldOrPropertyInitializer(original, rewritten, symbolIndex, createTemporary)
+        End Function
+
+        Public Overrides Function InstrumentForEachLoopInitialization(original As BoundForEachStatement, initialization As BoundStatement) As BoundStatement
+            Return Previous.InstrumentForEachLoopInitialization(original, initialization)
+        End Function
+
+        Public Overrides Function InstrumentForEachLoopEpilogue(original As BoundForEachStatement, epilogueOpt As BoundStatement) As BoundStatement
+            Return Previous.InstrumentForEachLoopEpilogue(original, epilogueOpt)
+        End Function
+
+        Public Overrides Function InstrumentForLoopInitialization(original As BoundForToStatement, initialization As BoundStatement) As BoundStatement
+            Return Previous.InstrumentForLoopInitialization(original, initialization)
+        End Function
+
+        Public Overrides Function InstrumentForLoopIncrement(original As BoundForToStatement, increment As BoundStatement) As BoundStatement
+            Return Previous.InstrumentForLoopIncrement(original, increment)
+        End Function
+
+        Public Overrides Function InstrumentLocalInitialization(original As BoundLocalDeclaration, rewritten As BoundStatement) As BoundStatement
+            Return Previous.InstrumentLocalInitialization(original, rewritten)
+        End Function
+
+        Public Overrides Function CreateUsingStatementPrologue(original As BoundUsingStatement) As BoundStatement
+            Return Previous.CreateUsingStatementPrologue(original)
+        End Function
+
+        Public Overrides Function InstrumentUsingStatementResourceCapture(original As BoundUsingStatement, resourceIndex As Integer, rewritten As BoundStatement) As BoundStatement
+            Return Previous.InstrumentUsingStatementResourceCapture(original, resourceIndex, rewritten)
+        End Function
+
+        Public Overrides Function CreateUsingStatementDisposePrologue(original As BoundUsingStatement) As BoundStatement
+            Return Previous.CreateUsingStatementDisposePrologue(original)
+        End Function
+
+        Public Overrides Function CreateWithStatementPrologue(original As BoundWithStatement) As BoundStatement
+            Return Previous.CreateWithStatementPrologue(original)
+        End Function
+
+        Public Overrides Function CreateWithStatementEpilogue(original As BoundWithStatement) As BoundStatement
+            Return Previous.CreateWithStatementEpilogue(original)
+        End Function
+    End Class
+
+End Namespace
```

Comment:
No, it is not the same. I believe there is enough language specific things to not try to unify at this point.